tests/python/multidevice/test_transformer.py

```diff
@@ -184,250 +134,237 @@ def __init__(self, num_devices, batch, sequence, head, hidden):
 
     def definition(self) -> None:
         # Same notations as in test_multidevice_transformer.cpp.
-        d, b, s, h, e = (
-            self._num_devices,
+        b, s, h, e = (
             self._batch,
             self._sequence,
             self._head,
             self._hidden,
         )
-
         self.inp = self.define_tensor(
             shape=[b, s, e],
-            contiguity=True,
+            contiguity=[True, True, True],
             dtype=DataType.BFloat16,
+            is_cpu=False,
+            stride_order=[2, 1, 0],
         )
         self.layernorm0_weight = self.define_tensor(
             shape=[e],
-            contiguity=True,
+            contiguity=[True],
             dtype=DataType.BFloat16,
+            is_cpu=False,
+            stride_order=[0],
         )
         self.layernorm0_bias = self.define_tensor(
             shape=[e],
-            contiguity=True,
+            contiguity=[True],
             dtype=DataType.BFloat16,
+            is_cpu=False,
+            stride_order=[0],
         )
         self.mha_linear0_weight = self.define_tensor(
-            shape=[d, e * 3 // d, e],
-            contiguity=True,
+            shape=[e * 3, e],
```

Comment:
I thought Thunder benchmarks use constant shapes. Did you make them symbols by yourself?